core/aws-core/src/main/java/software/amazon/awssdk/awscore/retry/AwsRetryPolicy.java

```diff
@@ -25,8 +25,11 @@
 
 /**
  * Retry Policy used by clients when communicating with AWS services.
+ *
+ * @deprecated Use instead {@link AwsRetryStrategy}
  */
 @SdkPublicApi
```

Comment:
Can we remove this @SdkPublicApi since its deprecated i think we can remove this

Reply:
I'm not sure that we can remove it, if I do the build will bark with:

```
SDK annotation is missing on this class. eg: @SdkProtectedApi [MissingSdkAnnotation]
```

If we do, which one should we use instead?

Reply:
Deprecated shouldn't change the public/protected/internalness
Confirmed this with the team.
Thanks for the patience while I was waiting to get clarification on this.